2024_acm_hpcsig_edu.md

```diff
@@ -0,0 +1,302 @@
+     style="width: 800px;"/>
+
+---
+
+<img src="img/2024-post-workshop-survey/reusable.png"
+     alt="Plot about whether code is more reusable"
+     style="width: 800px;"/>
+
+<img src="img/2024-post-workshop-survey/collaboration.png"
+     alt="Plot about whether collaboration is easier"
+     style="width: 800px;"/>
+
+---
+
+<img src="img/2024-post-workshop-survey/colleagues.png"
+     alt="Plot about whether colleagues have been introduced"
+     style="width: 700px;"/>
+
+<img src="img/2024-post-workshop-survey/recommending.png"
+     alt="How likely are you to recommend?"
+     style="width: 500px;"/>
+
+---
+
+# Collaboration across funding borders
+
+<img src="img/atc.jpg"
+     alt="Air traffic control tower"
+     style="height: 200px;"/>
+<img src="img/monitor-setup.jpg"
+     alt="Streaming setup during Python for Scientific Computing"
+     style="height: 200px;"/>
+
+0.9 FTE (2 persons) + [10 persons in-kind + volunteers](https://coderefinery.org/about/contributors/)
+
+<img src="img/CR_partners.png"
+     alt="CodeRefinery partner logos"
+     style="width: 750px;"/>
+
+Co-advertize and co-organize with us
+
+<img src="img/logos/tud.png"
+     alt="TU Delft logo"
+     style="height: 35px;"/>
+<img src="img/logos/nl-escience-center.png"
+     alt="The Netherlands eScience Center logo"
+     style="height: 35px;"/>
+<img src="img/logos/vu.png"
+     alt="VU Amsterdam logo"
+     style="height: 40px;"/>
+
+---
+
+# What we have learned
+
+## About motivating/teaching
+
+- Teaching isn't a lecture anymore. It's more
+  .emph[like a live TV production], which *can* be as interactive as people in
+  a room.
+
+- .emph[Co-teaching] is a great way to onboard, get better quality, and reduce
+  stress
+
+- .emph[Good enough practices] better than perfect practices not
+  applied
+
+- Instead of "good for others": ".emph[good for your future you] and as side
+  effect good for others"
+
+---
+
+# What we have learned
+
+## About scaling
+
+- "bring your own classroom" seems to be a way to scale
+
+- .emph[Installation instructions and on-boarding] become more important
+
+- We don't "see" classrooms -> .emph[feedback mechanism] in Q&A doc
+
+- Make exercises longer to .emph[give classrooms the chance to interact]
+
+---
+
+# Collaborative document: Markdown
+
+- Interactive, anonymous, parallel, async
+- New question every 1-2 minutes!
+- ASCII-graph feedback
+
+<img src="img/notes.png"
+     alt="Screenshot of exercise title and questions in collaborative notes"
+     style="width: 100%;" />
+
+We publish Q&A for each workshop: [Example](https://coderefinery.github.io/2024-03-12-workshop/questions/)
+
+---
+
+.left-column50[
+## Future: Community project
+
+- Communicate value for volunteers and organizations
+
+- Research groups send their students to us instead of
+  creating isolated material
+
+- More collaboration with similar projects ("helper exchange program")
+
+- Governance is community-driven
```

Comment:
```suggestion
- .emph[Communicate value] for volunteers and organizations

- Research groups send their students to us instead of
  creating isolated material

- .emph[More collaboration] with similar projects ("helper exchange program")

- Governance is .emph[community-driven]
```